validate-date-test/test.js

```diff
@@ -2,9 +2,18 @@ var test = require('tape');
 const validateDate = require('../validate-date');
 
 test('is valid date', function (t) {
-    t.plan(2);
+    
 
     t.equal(validateDate('02/29/2001'), 'Invalid Date');
-    t.equal(validateDate('02/2/9/2001'), 'Invalid Format');
-    t.equal(validateDate('02/29/2000'), 'Valid Date');
+    t.equal(validateDate('02/2/9/2001', boolean=false), 'Invalid Format');
+    t.equal(validateDate('02/29/2000', boolean=false), 'Valid Date');
+    t.equal(validateDate('02/29/2000', boolean=true), true);
+    t.equal(validateDate('02/29/2001',boolena=true), false);
```

Comment:
looks like a typo "boolena". Did this test succeed?

Reply:
It did succeed for some reason. Actually it seems not to matter what I name the parameter, just the order. Not sure what it is.... I will dig in.

Reply:
Ok....Just realized that JS does not care about the parameter names by default, just the order. So that is way it did succeed.